README.md

```diff
@@ -141,6 +142,15 @@ The following methods are exposed on an instantiated client
 | addActions | `(({ getNode, flat, dispatch, snapshot, extend, types, initNode }) => {actionsMethods} ) => null` | *Experimental* A method for extending the client with new actions on a per instance basis. You pass in a function which takes an object containing internal helpers and returns an object with actions that get extended onto the tree instance. |
 | addReactors | `(() => {customReactors}) => null` | *Experimental* A method for adding new reactors on a per instance basis. You pass in a function which returns an object of new reactors to support (`{reactorName: reactorFunction}`). Reactors are passed `(parent, node, event, reactor, types, lookup)` and are expected to return an array of affected nodes. |
 | subquery | `(targetPath, sourceTree, sourcePath, mapSubqueryValues?) => {}` | Sets up a subquery, using the types passed in to the client and assuming this tree instance is the target tree. For more info, see the [subquery](#Subquery) section below. |
+| replace | `async (path, node) -> await searchCompleted` | Replaces a the node at the given path with the new node. You can await this for when updates settle and relevant searches are completed. |
+| indent | `async (path, newNode) -> await searchCompleted` | Indents the node at the provided path by either `indentInPlace` or `indentReplace` depending on whether the target is the root node or not. You can await this for when updates settle and relevant searches are completed. |
+| indentReplace | `async (path, newNode) -> await searchCompleted` | Indents the node at the provided path by replacing it with a group described by `newNode` with the target node as its children. You can await this for when updates settle and relevant searches are completed. **NOTE:** This should rarely be used directly and is mostly an internal utility used by `indent`. You probably want `indent` which has the same signature. |
```

Comment:
I don't think the whole `{name} a node at the given path {action}` is necessary. It reads better like
```
Replace node at `path` with node `newNode` etc...
```
In general though I find this one confusing. Here's my take on it:
```
Let `oldNode` be subtree at `path`.
Then `replace` `oldNode` with `newNode` and set `oldNode` as `newNode`'s single child.
```

Reply:
The more I think about it, the more I think that these should probably be internal only methods and not actually part of the official API. There's really no reason to use `indentReplace` or `indentInPlace` over just `indent`

Reply:
So it occurs to me that while `indent` is the name of the feature in the QueryBuilder, this might be clearer if we call it something like `wrap` or `wrapInGroup` (and then `wrapInPlace`/`wrapReplace` or `wrapInGroupInPlace`/`wrapInGroupReplace`)

Reply:
I'm all in for less API surface. What about `{add,insert}Parent` for `indent`?